transformers/pipelines.py

```diff
@@ -94,32 +95,71 @@ def __call__(self, *texts, **kwargs):
 
 
 class QuestionAnsweringPipeline(Pipeline):
+    """
+    Question Answering pipeling involving Tokenization and Inference.
+    TODO:
+     - top-k answers
+     - return start/end chars
+     - return score
+    """
+
+    def __init__(self, model, tokenizer: Optional[PreTrainedTokenizer]):
+        super().__init__(model, tokenizer)
+
+    @staticmethod
+    def create_sample(question: Union[str, List[str]], context: Union[str, List[str]]) -> Union[dict, List[Dict]]:
+        is_list = isinstance(question, list)

```

Comment:
maybe accept list, tuples or any iterable of some kind?
Actually it's probably simpler to tests for `SIX.string_types` and accept any iterable of some kind if False.